particles/ManageProducts/ManageProducts.js

```diff
@@ -0,0 +1,172 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+"use strict";
+
+defineParticle(({DomParticle, resolver}) => {
+
+  let host = `[manage-products]`;
```

Comment:
This is basically same as ShowProducts, but with a "remove" button on each product.
I didn't want to change ShowProducts but I wanted to be able to test the remove functionality, so I added it, but I'm not sure it's worth being submitted here. WDYT is best?